packages/ember/index.js

```diff
@@ -791,6 +791,36 @@ if (IS_NODE) {
   context.exports.Ember = context.exports.Em = Ember;
 }
 
+if (window) {
+  Object.defineProperty(window, 'Ember', {
+    configurable: true,
+    writable: true,
+    get() {
+      deprecate(
+        "Using window.Ember has been deprecated. Use `import Ember  from 'ember';` instead ",
+        false,
+        {
+          id: 'ember-source.window-global',
+          until: '4.0.0',
+          url: 'tbd',
+          for: 'ember-source',
+          since: { enabled: '3.26.0-beta.1' },
+        }
+      );
+
+      Object.defineProperty(window, 'Ember', {
+        configurable: true,
+        writable: false,
+        get() {
+          return Ember;
+        },
+      });
+
+      return Ember;
+    },
+  });
+}
```

Comment:
I think this needs to be just above where `context.exports.Ember = context.exports.Em = Ember;` is.

`context.exports` _is_ `window` IIRC

Reply:
oh! this is good to know. thanks!